### DISCUSSION TITLE
[SUPPORT]  Allow certain specific cookies to be kept

### DISCUSSION THREAD

**Discussion by 5im-0n** [Cookie-AutoDelete/Cookie-AutoDelete#1000] [Q&A / Support] [answered]
**Is your feature request related to a problem? Please describe.**
I have set CAD to remove cookies from google.com. So every time I visit google.com a banner comes up, asking me to agree to something (Before you continue):
![google banner](https://i.vgy.me/e4Zmjw.png)
Once I agree, a cookie is set `CONSENT: YES+IT.en+V9+BX`, and the banner does not come up again.
It would be nice if I could tell CAD to keep only that specific cookie, so google does not bother me again with that banner.

**Describe the solution you'd like**
Add an option to CAD to keep specific cookies, even if every other cookie should be marked for deletion.

**Describe alternatives you've considered**
Keep clicking `I agree` on the banner every time I search for something on google, or allow google to set cookies :(

**Comment by kennethtran93** [comment, marked as answer]
Please read the documentation, more specifically this one:  https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/Documentation#whitelisting-individual-cookies

Simply uncheck the box to get a list of cookies currently set for that site/domain/expression, then check the one you want to keep.
